File: second_task/app/src/main/java/com/example/secondtask/main/MainActivity.java
package com.example.secondtask.main;

import androidx.annotation.NonNull;
import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.Fragment;

import android.os.Bundle;
import android.view.MenuItem;


import com.example.secondtask.R;
import com.example.secondtask.main.first_fragment.InputFragment;
import com.example.secondtask.main.second_fragment.ShowFragment;
import com.google.android.material.bottomnavigation.BottomNavigationView;

public class MainActivity extends AppCompatActivity {

    private BottomNavigationView navView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        navView = findViewById(R.id.nav_view);

        if(savedInstanceState==null){
            getSupportFragmentManager().beginTransaction().replace(R.id.fragment_view,new InputFragment()).commit();
        }
        setListeners();
    }

    public void setListeners(){
        navView.setOnNavigationItemSelectedListener(new BottomNavigationView.OnNavigationItemSelectedListener() {
            @Override
            public boolean onNavigationItemSelected(@NonNull MenuItem item) {
                Fragment selected = null;
                switch (item.getItemId()){
                    case R.id.input_menu:
                        selected = new InputFragment();
                        break;
                    case R.id.show_menu:
                        selected = new ShowFragment();
                        break;
                }

                getSupportFragmentManager().beginTransaction().replace(R.id.fragment_view,selected).commit();
                return true;
            }
        });
    }
}

File: second_task/app/src/main/java/com/example/secondtask/main/MainActivity.kt
package com.example.secondtask.main

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.fragment.app.Fragment
import com.example.secondtask.R
import com.example.secondtask.databinding.ActivityMainBinding
import com.example.secondtask.main.first_fragment.InputFragment
import com.example.secondtask.main.second_framgment.ShowFragment
import com.google.android.material.bottomnavigation.BottomNavigationMenu

class MainActivity : AppCompatActivity() {

    private lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        val view = binding.root
        setContentView(view)

        if(savedInstanceState==null)
            setCurrentFragment(InputFragment())

        setListeners()

    }

    fun setListeners(){
        binding.navView.setOnNavigationItemSelectedListener {
            when(it.itemId){
                R.id.input_menu -> setCurrentFragment(InputFragment())
                R.id.show_menu -> setCurrentFragment(ShowFragment())
            }
            true
        }
    }

    fun setCurrentFragment(fragment : Fragment){
        supportFragmentManager.beginTransaction().apply {
            replace(binding.fragmentView.id,fragment)
            commit()
        }
    }


}